doc/postgresql.md

```diff
@@ -0,0 +1,18 @@
+# Cloud Resource Operator - PostgreSQL
+
+## Usage
+To seed a Kubernetes/Openshift cluster with an example Postgres resource:
+```
+$ make cluster/prepare 
+$ make cluster/seed/postgres
+```
+
+## Strategy 
+A config map object is expected to exist for each provider that will be used by the operator. This config map contains provider-specific information about how to deploy a particular resource type, such as blob storage. In the Cloud Resources Operator, this provider-specific configuration is called a strategy. An example of the AWS strategy can be seen [here](../deploy/examples/config_aws_strat.yaml)
+
+### Kubernetes/Openshift Strategy
+For Kubernetes/Openshift a JSON object containing a single key `strategy`, under strategy is the following keys which are used to overwrite the object specs:
+- [PostgresDeploymentSpec](https://godoc.org/k8s.io/api/apps/v1#DeploymentSpec)
+- [PostgresServiceSpec](https://godoc.org/k8s.io/api/core/v1#ServiceSpec)
+- [PostgresPVCSpec](https://godoc.org/k8s.io/api/core/v1#PersistentVolumeClaimSpec)
+- PostgresSecretData - A `map[string]string` with the key `redis.conf` 
```

Comment:
You can remove the `redis.conf`, currently the keys supported for postgres are `user`, `password`, `database`